manage_app.py

```diff
@@ -236,35 +333,70 @@ def check_service_up(url: str, service: str, discord=False):
     """
 
     print()
-    print(bcolors.YELLOW_IN +
-          "CHECKING SERVICE {}:{} ".format(ENV, service) + bcolors.CEND)
+    print(
+        bcolors.YELLOW_IN +
+        "CHECKING SERVICE {}:{} ".format(
+            os.getenv("DOCK_ENV"), service,
+        ) + bcolors.CEND,
+    )
 
     req_method = ""
 
     if service == "WEB":
         try:
-            web_request = requests.get(url)
+            web_request = requests.get(
+                url, verify=False if ENV != "PROD" else True,
```

Comment:
* Why the ternary?
* `verify=ENV != "LOCAL"`
* All deployed environments have certificates and we want to make verified requests to check their status